### PR TITLE
Add Support for Ktuvit

### PR DESCRIPTION
**Pull request by GuyKh**
https://www.ktuvit.me is a Hebrew Subtitles website

### CODE REVIEW

**Comment by GuyKh** [comment]
Don't merge just yet.

**Comment by morpheus65535** [comment]
I'll wait for your confirmation before reviewing this PR.

**Review by LASER-Yi — APPROVED** [review]

File: libs/subliminal_patch/providers/ktuvit.py [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+        self.email = email
+        self.hashed_password = hashed_password
+        self.logged_in = False
+        self.session = None
+        self.loginCookie = None
+        self.headers = None
+
+    def initialize(self):
+        self.session = Session()
+        self.session.headers["User-Agent"] = "Subliminal/{}".format(__short_version__)
```
Comment (morpheus65535) [inline libs/subliminal_patch/providers/ktuvit.py:128]: You should use Bazarr user-agent. See OpenSubtitles as example.

File: libs/subliminal_patch/providers/ktuvit.py [hunk 2]
```diff
@@ -0,0 +1,440 @@
...
+            logger.info("Logging in")
+
+            data = {"request": {"Email": self.email, "Password": self.hashed_password}}
+            headers = {
+                'Accept-Encoding': 'gzip',
+                'Accept-Language': 'en-us,en;q=0.5',
+                'Pragma': 'no-cache',
+                'Cache-Control': 'no-cache',
+                'Content-Type': 'application/json',
+                'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.97 Safari/537.36',
```
Comment (morpheus65535) [inline libs/subliminal_patch/providers/ktuvit.py:141]: Why are you overwriting user-agent here?

**Review by morpheus65535 — COMMENTED** [review]
Other than that it looks good.

File: libs/subliminal_patch/providers/ktuvit.py (hunk 2 above)
Comment (GuyKh) [reply libs/subliminal_patch/providers/ktuvit.py:141]: 👍🏻

File: libs/subliminal_patch/providers/ktuvit.py (hunk 1 above)
Comment (GuyKh) [reply libs/subliminal_patch/providers/ktuvit.py:128]: 👍🏻

**Comment by GuyKh** [comment]
@morpheus65535 - feel free to review :)

File: libs/subliminal_patch/providers/ktuvit.py [hunk 3]
```diff
@@ -418,7 +418,7 @@ def download_subtitle(self, subtitle):
             # download
             url = self.server_url + self.request_download_id_url
             r = self.session.post(
-                url, data=json.dumps({"request": downloadIdentifierRequest}), timeout=10
+                url, data={"request": downloadIdentifierRequest}, timeout=10
```
Comment (morpheus65535) [inline libs/subliminal_patch/providers/ktuvit.py:421]: Should it be json instead of data?

Comment (GuyKh) [reply libs/subliminal_patch/providers/ktuvit.py:421]: good catch.
Fixed

**Pull request merged**